### DISCUSSION TITLE
HTB CPTS ranking

### DISCUSSION THREAD

**Discussion by Miliik** [PaulJerimy/SecCertRoadmapHTML#142] [General]
Hi everyone, 

I'm currently a Cybersecurity student in my PHD last year, and in my school we love your roadmap as it helped us a lot to choose our next Cert. As part of my cursus we have the possibility to pass some Cert for a grade, and as me and friends passed the CPTS and OSCP, we are really suprise to see that cpts is so low rank compared to OSCP, when for the exam we globally felt CPTS being 2 times harder than OCSP (we both follow the courses proposed by the two cert). Is there a Reason for the ranking we don't think of ?

**Comment by jwctexas210** [comment]
I have noticed this too. Paul needs to pass some of these HTB certs before putting them on the bottom of his shoe. I have CYSA GSEC and CDSA. It’s like elementary school vs college. 
HTB is criminally underrated on his chart.
